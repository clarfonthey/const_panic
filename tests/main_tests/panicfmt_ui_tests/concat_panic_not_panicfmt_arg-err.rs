struct NotFmt;


const _: () = const_panic::concat_panic!(NotFmt);

fn main() {}