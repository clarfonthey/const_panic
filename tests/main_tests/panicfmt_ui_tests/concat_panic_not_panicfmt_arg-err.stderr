error[E0277]: the trait bound `NotFmt: PanicFmt` is not satisfied
 --> tests/main_tests/panicfmt_ui_tests/concat_panic_not_panicfmt_arg-err.rs:4:15
  |
4 | const _: () = const_panic::concat_panic!(NotFmt);
  |               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `PanicFmt` is not implemented for `NotFmt`
  |
  = help: the following other types implement trait `PanicFmt`:
            &'a T
            ()
            *const T
            *mut T
            ArrayString<CAP>
            Delimiter
            FromBytesUntilNulError
            FromBytesWithNulError
          and $N others
  = note: this error originates in the macro `$crate::coerce_fmt` which comes from the expansion of the macro `const_panic::concat_panic` (in Nightly builds, run with -Z macro-backtrace for more info)
